buildchow/templates/core-jobs.yaml

```diff
@@ -5,9 +5,11 @@
       master
     devstudio-stream:
       master
-    target-platform-version: 4.50.0.Beta1-SNAPSHOT
-    target-platform-version-maximum: 4.50.0.Beta1-SNAPSHOT
-    disabled: true
+    target-platform-version: 4.50.0.CR1-SNAPSHOT
+    target-platform-version-maximum: 4.50.0.CR1-SNAPSHOT
+    blockBuildWhenDownstreamBuilding: false
```

Comment:
interesting. Do we really want this ?

I'm not sure, but one advantage having this true is
that if you push two related changes (i.e. one in base and one in openshift) you don't see openshift fail because it is missing the change from base ?